### DISCUSSION TITLE
出现错误反馈，没有权限

### DISCUSSION THREAD

**Discussion by CYSLK** [Meloong-Git/PCL#2020] [提问] [answered]
提示权限不足，可我的账户在电脑上是管理员，【以管理员身份运行】也不行
<img width="320" alt="屏幕截图 2023-06-10 204012" src="https://github.com/Hex-Dragon/PCL2/assets/136172729/eb22d955-f747-4dd2-9622-701fc165ad56">

**Comment by WForst-Breeze** [comment, marked as answer]
请前往 Issue 界面提交 [综合 Bug 反馈](https://github.com/Hex-Dragon/PCL2/issues/new?assignees=&labels=%C2%B7+Bug%2C%E6%96%B0%E5%8F%8D%E9%A6%88&projects=&template=bug9.yml)，并上传相关日志及复现信息

**Reply by CYSLK** [reply]
我不知道怎么再现啊，第二天就好了……

**Comment by so1ve** [comment]
锟斤拷烫烫烫=。=
